lib/mocha.js

```diff
@@ -311,6 +314,43 @@ Mocha.prototype.loadFiles = function(fn) {
   fn && fn();
 };
 
+/**
+ * Loads `files` prior to execution. Supports Node ES Modules.
+ *
+ * @description
+ * The implementation relies on Node's `require` and `import` to execute
+ * the test interface functions and will be subject to its cache.
+ * Supports both CJS and ESM modules.
+ *
+ * @private
+ * @see {@link Mocha#addFile}
+ * @see {@link Mocha#run}
+ * @see {@link Mocha#unloadFiles}
+ *
+ * @returns {Promise}
+ */
+Mocha.prototype.loadFilesAsync = function() {
+  var self = this;
+  var suite = this.suite;
+
+  if (!esmUtils) {
```

Comment:
I'm unsure wether `esmUtils` is working correctly.
Our CI tests do not enter `if (!esmUtils) {`, not one single time. That's weird.

Reply:
Coveralls seems to run only with Node v13, so `esmUtils` is never false.